### ISSUE TITLE
A less dark dark theme?

### ISSUE THREAD

**Issue by muchqs** [AdguardTeam/AdGuardHome#5465]
### Prerequisites

- [X] I have checked the [Wiki](https://github.com/AdguardTeam/AdGuardHome/wiki) and [Discussions](https://github.com/AdguardTeam/AdGuardHome/discussions) and found no answer

- [X] I have searched other issues and found no duplicates

- [X] I want to request a feature or enhancement and not ask a question

#### What problem are you trying to solve?
The almost-black color background in the new dark theme is a bit too dark.
Almost black with almost white fonts is too much contrast and actually not that easy on the eyes.

#### Proposed solution
A dark shade of grey is better.
Or, dark shade of blue with white-ish but not pure white letters is better.

**Comment by ammnt** [comment]
I agree. Need more UI/UX improvements here. I create the userscript based on Dracula color scheme:
https://en.wikipedia.org/wiki/Dracula_(color_scheme)

It's a little softer for me:
![image](https://user-images.githubusercontent.com/45385632/218174830-77db49a7-0dcf-4754-83b6-f2551076d49b.png)

![image](https://user-images.githubusercontent.com/45385632/218174929-cde4d72c-d144-4c55-bb67-a9896b3d4d29.png)
